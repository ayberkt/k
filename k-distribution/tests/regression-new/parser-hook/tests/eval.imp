int a;
a = eval("eval(\"1+2\")") + eval("2+3");