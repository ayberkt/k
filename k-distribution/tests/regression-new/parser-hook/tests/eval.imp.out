<T> <k> . </k> <state> a |-> 8 </state> </T>